src/client.py

```diff
@@ -44,3 +44,19 @@ def get_robot_state(self) -> str:
             return
 
         return response_text
+
+    def set_true_camera_action_skip(self) -> bool:
+        """撮影終了フラグを立てる.
+
+        Returns:
+            success (bool): 通信が成功したか(成功:true/失敗:false)
+        """
+        url = f"http://{self.server_ip}/"
```

Comment:
```suggestion
        url = f"http://{self.server_ip}/robot_info/skip_camera_action_true"
```
理解できていなかったらごめん。走行体側でこのURLって言ってた気がするんだけど、それでいいんだっけ？

Reply:
よくないです